src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4460,7 +4576,8 @@ private MemberAst ClassMemberRule(string className, out List<Ast> astsOnError)
             if (lastAttribute != null)
             {
                 // We have the start of a member, but didn't see a variable or 'def'.
-                ReportIncompleteInput(After(ExtentFromFirstOf(lastAttribute)),
+                ReportIncompleteInput(
+                    After(ExtentFromFirstOf(lastAttribute)),
```

Comment:
@IISResetMe Please move all formatting changes in separate PR. Our policy is avoid mixing style and functional changes.

Reply:
Ah, misunderstood your previous comment then

Reply:
My request was only for your new or changed code but if you fixed all code please move this to new PR and keep current PR as small as possible.

Reply:
Done, will fix the indentation style issues in a separate PR